src/templates/mocks/mapAndLabelMocks.ts

```diff
@@ -0,0 +1,212 @@
+// Example A is based on schema "Sketch plan - TPO" with 2 features in Southwark
+export const mapAndLabelOutputA: GeoJSON.FeatureCollection = {
+  type: "FeatureCollection",
+  features: [
+    {
+      type: "Feature",
+      geometry: {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-0.07627111599063874, 51.485963105125705],
+            [-0.07630330249881746, 51.48592886542687],
+            [-0.07627513930416108, 51.48590631731852],
+            [-0.07622149512386324, 51.4859096577797],
+            [-0.07622417733287812, 51.48593972191955],
+            [-0.07627111599063874, 51.485963105125705],
+          ],
+        ],
+      },
+      properties: {
+        label: "1",
+        "area.squareMetres": 23.37,
+        "area.hectares": 0.002337,
+        species: "Elm",
+        work: "Trim branches",
+        reason:
+          "A couple of the branches at the bottom of the trunk have died and should be trimmed for safety before they fall. Healthy branches at the top will remain untouched.",
+      },
+    },
+    {
+      type: "Feature",
+      geometry: {
+        type: "Polygon",
+        coordinates: [
+          [
+            [-0.07581782266712189, 51.48591800893166],
+            [-0.07586342022037507, 51.485876253156675],
+            [-0.07580172941303254, 51.485851199673306],
+            [-0.07574406191921236, 51.48589629593343],
+            [-0.07580172941303254, 51.48592134939199],
+            [-0.07581782266712189, 51.48591800893166],
+          ],
+        ],
+      },
+      properties: {
+        label: "2",
+        "area.squareMetres": 34.23,
+        "area.hectares": 0.003423,
+        species: "Oak",
+        work: "Trim branches",
+        reason:
+          "Some branches are overgrowing into the neighbours garden beyond my property curtilege",
+      },
+    },
+  ],
+};
+
+export const mapAndLabelNodePropsA: any = {
```

Comment:
I've intentionally mocked "NodeProps" to match the full new breadcrumb data I'm submitting in planx-new, even though we ultimately only need to rely on a couple of these props to generate the doc ! Makes it a little easier to iterate / understand universe of what's available.